### PR TITLE
python3-platformdirs: update to 3.5.1.

### PR DESCRIPTION
**Pull request by mhmdanas**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/python3-calver/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+license="Apache-2.0"
+homepage="https://github.com/di/calver"
+changelog="https://github.com/di/calver/releases"
+distfiles="${PYPI_SITE}/c/calver/calver-${version}.tar.gz"
+checksum=e05493a3b17517ef1748fbe610da11f10485faa7c416b9d33fd4a52d74894f8b
+make_check=no # PyPI tarball has no tests.
+
+pre_patch() {
+	vsed -e "s/version=calver_version(True),/version=\"$version\"/" -i setup.py
+}
```
Comment (ahesford) [inline srcpkgs/python3-calver/template:19]: Does the automated version detection fail to work as they expect?

**Comment by tornaria** [comment]
I'm trying to update jupyter packages and this would help me.

I've no idea of calver... the reason why hatchling needs updating is because platformdirs demands it, but it builds fine with the current version so I just patched it out.

However, I guess it's better in the long term to have  hatchling updated. It seems jupyter packages moved to build with hatchling.

File: srcpkgs/python3-calver/template (hunk 1 above)
Comment (ahesford) [reply srcpkgs/python3-calver/template:19]: I did some tests and this package seems to be properly versioned without this patch. Am I missing something here?

Comment (mhmdanas) [reply srcpkgs/python3-calver/template]: Indeed, I just checked myself and this line seems unnecessary. I'm sure I added this for a reason, but I can't remember what that was now...

Anyway, removed it.

**Pull request merged**